Missing package undent

When running the pluging I have the error
```
[03:23:08:0650 PM]  Error when loading plugin 'rooibos-roku': Error: Cannot find module 'undent'
```
installing (explicitly) undent fixed the problem, but this should be marked as dependency in the package

--- a/bsc-plugin/package.json
+++ b/bsc-plugin/package.json
@@ -15,6 +15,7 @@
     },
     "dependencies": {
         "source-map": "^0.7.3",
+        "undent": "^0.1.0",
         "vscode-languageserver": "~6.1.1",
         "vscode-languageserver-protocol": "~3.15.3"
     },
@@ -39,8 +40,7 @@
         "source-map-support": "^0.5.13",
         "trim-whitespace": "^1.3.3",
         "ts-node": "^9.0.0",
-        "typescript": "^3.8.3",
-        "undent": "^0.1.0"
+        "typescript": "^3.8.3"
     },
     "scripts": {
         "preversion": "npm run build && npm run lint && npm run test",

